spec/features/staff/fund_managers_can_edit_an_organisation_spec.rb

```diff
@@ -1,27 +1,28 @@
 RSpec.feature "Fund managers can edit organisations" do
-  let!(:organisation) { create(:organisation) }
+  let!(:beis_organisation) { create(:organisation) }
+  let!(:another_organisation) { create(:organisation) }
 
   context "when the user is not logged in" do
     it "redirects the user to the root path" do
       page.set_rack_session(userinfo: nil)
-      visit edit_organisation_path(organisation)
+      visit edit_organisation_path(beis_organisation)
       expect(current_path).to eq(root_path)
     end
   end
 
-  context "when the user is allowed to edit an organisation" do
+  context "when the fund manager is allowed to edit an organisation" do
     scenario "successfully editing an organisation" do
-      authenticate!(user: build_stubbed(:fund_manager))
+      authenticate!(user: create(:fund_manager, organisation: beis_organisation))
 
       successfully_edit_an_organisation
```

Comment:
It looks like `successfully_edit_an_organisation` is only used here and it relies on the two organisations, I wonder if just including it here would make the spec easier to read?